### PR TITLE
feat(sdk): config to control webview hardware acceleration

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #110

### CODE REVIEW

**Comment by AAMARAB** [comment]
طلبي

File: README.md [hunk 1]
```diff
@@ -151,29 +151,30 @@ To release all resources you may call `HCaptcha.reset()`.
...
+| `hideDialog`                  | Boolean                 | No       | False                            | To be used in combination with a passive sitekey when no user interaction is required. See Enterprise docs.                                                          |
+| `tokenExpiration`             | long                    | No       | 120                              | hCaptcha token expiration timeout (seconds).                                                                                                                         |
+| `diagnosticLog`               | Boolean                 | No       | False                            | Emit detailed console logs for debugging                                                                                                                             |
+| `disableHardwareAcceleration` | Boolean                 | No       | True                             | Disable WebView hardware acceleration                                                                                                                                |
```
Comment (DSergiu) [inline README.md:177]: Please bump version and add a changelog.

Comment (CAMOBAP) [reply README.md:177]: Done

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+# 3.8.3
```
Comment (DSergiu) [inline CHANGES.md:3]: Should be `3.9.0`.

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
+
+- Feature: add config to control WebView hardware acceleration
```
Comment (DSergiu) [inline CHANGES.md:5]:
```suggestion
# 3.9.0

- Feature: add config to control WebView hardware acceleration `HCaptchaConfig.disableHardwareAcceleration`
- Fix: removed unsafe cast with improved public api
```

File: sdk/build.gradle [hunk 4]
```diff
@@ -22,11 +22,11 @@ android {
         // See https://developer.android.com/studio/publish/versioning
         // versionCode must be integer and be incremented by one for every new update
         // android system uses this to prevent downgrades
-        versionCode 34
+        versionCode 35
 
         // version number visible to the user
         // should follow semantic versioning (See https://semver.org)
-        versionName "3.8.2"
+        versionName "3.8.3"
```
Comment (DSergiu) [inline sdk/build.gradle:29]:
```suggestion
        versionName "3.9.0"
```

**Review by DSergiu — APPROVED** [review]

**Pull request merged**